UT4MasterServer.Models/DTO/Response/MMRRatingResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Text.Json.Serialization;
+
+namespace UT4MasterServer.Models.DTO.Response;
+
+public sealed class MMRRatingResponse
+{
+	[JsonPropertyName("rating")]
+	public int Rating { get; set; }
+
+	[JsonPropertyName("numGamesPlayed")]
+	public int GamesPlayed { get; set; }
```

Comment:
give explicit default value 0

Reply:
Int values are 0 by default.

Reply:
Did that in the constructor of the response object.